website/docs/learning-and-resources/how-to-guides/README.md

```diff
@@ -8,15 +8,16 @@ This document presumes you are familiar with the [core concepts](core-concepts/c
 
 ## Guide Reference
 
-1. [OAuth2 authorization for Google Sheets](oauth2-authorization-for-google-sheets.md)
```

Comment:
@danciaclara - Should we have cards for these too? We can do a separate PR for this.